deployment/legacy-migration/maintenance-worker/migration/neo4j/users/delete.cql

```diff
@@ -1 +1,2 @@
-MATCH (n:User) DETACH DELETE n;
\ No newline at end of file
+MATCH (n:User) DETACH DELETE n;
+MATCH (e:EmailAddress) DETACH DELETE e;
```

Comment:
code creep?

Reply:
No, you get a unique constrain violation error if you run the database import multiple times. @ulfgebhardt deletes each label separately instead of removing all at once (including in this case `EmailAddress`)